data/sql/updates/pending_db_world/rev_1725625860N.sql

```diff
@@ -0,0 +1,61 @@
+-- Script to add missing spanish options in some city guardians directions
+INSERT INTO acore_world.gossip_menu_option_locale VALUES
```

Comment:
never include database name when you do a PR, moreover use the backtick ` and add the related name field behind `VALUES`, like (`field1`, `field2`, `field3`, `field4`, `field5`)
```suggestion
INSERT INTO `gossip_menu_option_locale` VALUES
```